vision/transforms/transforms.go

```diff
@@ -13,3 +18,53 @@ type NormalizeTransformer struct {
 func Normalize(mean float64, stddev float64) *NormalizeTransformer {
 	return &NormalizeTransformer{mean, stddev}
 }
+
```

Comment:
Maybe put `Normalize` function in a seperated file, and `transforms.go` contains only interfaces.

Reply:
I'm not sure, the code of `Normalize` is short, and other functions of Tensor e.g. `Add/Mul/...` are all packaged in `tensor.go`, maybe we can separate it from `transforms.go` if we have many transformers?